index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PROJECT REVIEW</title>
+</head>
+<body>
+    <main id="main">
+        <!--  header and body section -->
+        <h1 id="title">Chris Brown biography</h1> 
+        <div id="img-div">
+          <img src="https://citas.in/media/authors/chris-brown.jpeg" alt="photo" id="image">
+          <p id="img-caption">Chris Brown in concert</p></div>
+            <p id="tribute-info">
+            Christopher Maurice Brown (born May 5, 1989) is an American singer, rapper, songwriter, dancer, and actor. Born in Tappahannock, Virginia, he was involved in his church choir and several local talent shows from a young age. Having signed with Jive Records in 2004, Brown released his self-titled debut studio album the following year, which became certified double platinum by the Recording Industry Association of America (RIAA)</p>
+            <!-- end of header and body section -->
+            <!-- link -->
+            <a target="_blank" rel="noopener noreferrer" id ="tribute-link" href="https://www.chrisbrownworld.com/home/">Chris Brown's Official web-page</a>
+    </main>
+            <!-- end link -->
+    </body>
+</html>
```

Comment:
### Wow, you've done an amazing job🦾🦾🦾, However, I will suggest you add a new line after your code😞😟😞.
[ ] - Adding a new line helps your code to be:

1. Neat😍
2. Follow semantics HTML and CSS rules😎😎
3. Readable✌✌.
3. 

-